pyproject.toml

```diff
@@ -62,6 +62,7 @@ dask-worker = "distributed.cli.dask_worker:main"
 [tool.setuptools]
 include-package-data = true
 zip-safe = false
+package-dir = {"" = "."}
```

Comment:
this causes setuptools to go down the same codepath as `is_compat_mode`:

https://github.com/pypa/setuptools/blob/e94cb204693cee145ea531905dd50b02a26c9f0a/setuptools/command/editable_wheel.py#L376-L378

```
        if set(self.package_dir) == {""} and has_simple_layout or is_compat_mode:
            # src-layout(ish) is relatively safe for a simple pth file
            src_dir = self.package_dir.get("", ".")
```